.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
It's an unlikely edge case, but if we tagged "very_important_milestone", this would trigger this workflow.  If I'm reading the docs right, a pattern of `'v[0-9]+.[0-9]+*'` would give us at least some superficial protection against this.

Reply:
I think the risk here and for the next comment is low. If an extra tag was pushed, it would just create a draft release. We would just need to go to the release tab and click on the delete draft button to clean it up. All the tags in the repo right now are for releases. I could add in the regular expression, but might be overkill to prevent an extra draft release for some one off event.

Reply:
Fair enough. As long as it's easy to clean up drafts that get mistakenly created, I agree that the extra complexity probably isn't worth it.

Reply:
Yup, it is very easy to clean them up, a single button click. :+1: